app/controllers/application_controller.rb

```diff
@@ -22,4 +22,8 @@ def current_user
     User.find(session[:user]) if session[:user]
   end
 
+  def grand_total(order_items)
+    order_items.reduce(0){|sum, cart_item| sum += cart_item.total}
```

Comment:
Useless assignment to variable - `sum`. Use just operator `+`.<br>Prefer `inject` over `reduce`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.